### PR TITLE
[#4, #5] fix student list css

### PR DESCRIPTION
**Pull request by kammanz**
[#4] Fixed student card bug, colors and font sizes.  

[#5] Fixed font colors and sizes. Please see note, as you wrote that I should change the search tag input color to light grey, but on the design the color appears as dark grey, so I changed it to that instead.

### CODE REVIEW

File: src/components/StudentCard.css [hunk 1]
```diff
@@ -63,8 +63,9 @@
 .button-container button {
     background: white;
     border: none;
-    font-size: 50px;
-    color: #A09FA1;
+    font-size: 70px;
+    font-weight: 600;
+    color: #757575;
```
Comment (Tumne) [inline src/components/StudentCard.css]: Noticed a blue outline around the button:

Screen Shot 2019-11-14 at 11 06 33 AM

add `outline: none;` here

File: src/components/StudentCard.css [hunk 2]
```diff
@@ -42,7 +42,7 @@
 
 .info-container form input:focus {
     outline: none;
-    border-bottom: 2px solid #A09FA1;
+    border-bottom: 1px solid black;
```
Comment (Tumne) [inline src/components/StudentCard.css]: There should be a space between details and grades:

Screen Shot 2019-11-14 at 11 10 04 AM

Also add more space between grade labels and grades percentage.

File: src/components/StudentCard.css (hunk 1 above)
Comment (Tumne) [inline src/components/StudentCard.css]: Another design thing, Add a tag should have a bolder underline. See design.

File: src/components/Students.css [hunk 3]
```diff
@@ -16,23 +16,28 @@
...
+    color: #757575;
+    border-bottom: 1px solid black;
 }
 
 #input-container input {
     width: calc(100% - 20px);
     margin: 0 10px;
     border-bottom: 1px solid #d7d6d7;
-    color: #A09FA1;
+    color: #757575;
```
Comment (Tumne) [inline src/components/Students.css]: Add `font-size: 20px;` here.

File: src/components/StudentCard.css (hunk 2 above)
Comment (kammanz) [reply src/components/StudentCard.css]: add white space not working, tried space bar and {" "}

Comment (Tumne) [reply src/components/StudentCard.css]: The simple way is to add `&nbsp;` instead

Comment (Tumne) [reply src/components/StudentCard.css]: `Test {index + 1}:&nbsp;&nbsp;&nbsp;{grade}%

File: src/components/StudentCard.css [hunk 4]
```diff
@@ -63,8 +63,9 @@
 .button-container button {
     background: white;
     border: none;
-    font-size: 50px;
-    color: #A09FA1;
+    font-size: 70px;
```
Comment (Tumne) [inline src/components/StudentCard.css]: Can we change this to 
`font-size: 60px;`

File: src/components/StudentCard.css (hunk 2 above)
Comment (kammanz) [reply src/components/StudentCard.css]: worked. damn son you got an eagles eye

Comment (Tumne) [reply src/components/StudentCard.css]: 🦅 👁

File: src/components/StudentCard.css (hunk 4 above)
Comment (kammanz) [reply src/components/StudentCard.css]: i made this change already maybe it didnt get included in my last push

Comment (Tumne) [reply src/components/StudentCard.css]: I know you did, let's just stick to this PR and ss4 branch

**Comment by Tumne** [comment]
Can you line up the plus/minus sign to the top?

Screen Shot 2019-11-14 at 12 38 03 PM

File: README.md [hunk 5]
```diff
@@ -1,68 +1,23 @@
...
 
-In the project directory, you can run:
+Built With
+----------
+React, create-react-app, Google Fonts
 
-### `npm start`
+Installation
+------------
+NPM install, NPM start
```
Comment (Tumne) [inline README.md]: can we format this section to the following:

Installation
------------
To install: 
```
npm install
```

To run:
```
npm start
```

File: README.md [hunk 6]
```diff
@@ -1,68 +1,23 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Student Search App
+------------------
+This application produces an array of students and their personal information, such as email address, employer, defining skill and average score over eight tests. The user can view the student's individual test scores by clicking a dropdown button. They can also add tags to the student. If they want to search the database, they can filter by first name, last name or tag. Results will autofill as the user types.
 
-## Available Scripts
+Description
+-----------
+Have completed steps 1 through 5, breaking up the tasks into two React components, a class component for controlling state and a functional component for rendering the student's info. I have used react useState and useEffect hooks in the past, but decided to keep it "old schoool" for this project as many jobs still use Raect the traditional way.
```
Comment (Tumne) [inline README.md]: Typo. `Raect` should be `React` (near end of paragraph)

File: README.md [hunk 7]
```diff
@@ -1,68 +1,23 @@
...
-### `npm start`
+Installation
+------------
+NPM install, NPM start
 
-Runs the app in the development mode.<br>
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+Authors
+-------
+Kaumil Manzoor
```
Comment (Tumne) [inline README.md]: also put your github here:

`Kaumil Manzoor - <your github link>

**Pull request merged**